docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -79,8 +81,19 @@ Preparing For a Release
    This should be an "Improvement" in the current sprint, with "Release Flocker $VERSION" as the title, and it should be assigned to yourself.
    The issue does not need a design, so move the issue to the "Coding" state.
 
+#. If this is a maintenance release, announce on Zulip's Engineering > Maintenance Release stream that a maintenance release is in progress.
+
+   ::
+
+      @engineering I am releasing from release/flocker-0.3.2. Please don't land anything on that branch until the release is complete.
```

Comment:
:question: Perhaps explain briefly what needs to be done to check that nothing has landed when it shouldn't, and what to do if something has.